cpmpy/tools/dimacs.py

```diff
@@ -72,41 +72,43 @@ def write_dimacs(model, fname=None):
 def read_dimacs(fname):
     """
         Read a CPMpy model from a DIMACS formatted file
```

Comment:
Maybe add a link to the DIMACS specification here to say we adhere to the strict specification and assume the file is well formed (that is with a header line)